src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -299,6 +299,30 @@ def test_pf_flow_test(self):
             log_content = f.read()
         assert previous_log_content not in log_content
 
+        # Test flow test with simple input file
+        run_pf_command(
+            "flow",
+            "test",
+            "--flow",
+            f"{FLOWS_DIR}/web_classification",
+            "--inputs",
+            f"{DATAS_DIR}/webClassification1.jsonl",
+        )
+        output_path = Path(FLOWS_DIR) / "web_classification" / ".promptflow" / "flow.output.json"
+        assert output_path.exists()
+
+        # Test flow test with invalid simple input file
+        with pytest.raises(ValueError) as ex:
```

Comment:
add a test on invalid file extension passed in

Reply:
add test case